### DISCUSSION TITLE
Student Developer Pack Application

### DISCUSSION THREAD

**Discussion by Sar-thak-3** [community/community#59438] [GitHub Education]
### Hi everyone,

I am new to GitHub Octernships, so I followed the procedure to apply for the same. My application for the student developer pack is approved but the changes aren't reflected in the past many days. I still see the same page " Seems you are not eligible for Octernships ".

What might be the reason? Thanks in advance!

**Comment by Shr11** [comment]
Hey Sarthak, I guess you are lacking in Contributions but no worries just find a niche & start contributing .Also, make sure to ensure all the below points if you haven't already:

Becoming a verified student on GitHub Global Campus.
Becoming an active contributor on GitHub.
Enrolling in an institution with work eligibility in one of the ten countries: India, Singapore, Indonesia, Malaysia, Vietnam, Philippines, Thailand, Mexico, Nigeria, and Colombia.
Learning about open source software development.
Practicing your coding skills.

**Comment by faridnec** [comment]
you need to contribute to an open source i'd say.
